bigquery_magics/bigquery.py

```diff
@@ -100,6 +100,7 @@
 from google.cloud.bigquery import exceptions
 from google.cloud.bigquery.dbapi import _helpers
 
+import bigquery_magics
```

Comment:
This is a little circular. Could you please import `bigquery_magics.version` instead to get this?

Reply:
Done.